workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/client-extension.dev.yaml

```
@@ -24,6 +24,11 @@ liferay-marketplace-custom-element-published-apps:
         - http://localhost:3000/@vite/refresh.ts
         - http://localhost:3000/src/main.tsx
 liferay-marketplace-custom-element-purchased-apps:
+    urls:
+        - http://localhost:3000/@vite/client
+        - http://localhost:3000/@vite/refresh.ts
+        - http://localhost:3000/src/main.tsx
+liferay-marketplace-custom-element-purchased-get-app:
```

```suggestion
liferay-marketplace-custom-element-purchased-solutions:
```

We already have a client extension called get app. Let's try to find a different name for this new one.